### ISSUE TITLE
When you refresh the page with post, visits counter increases

### ISSUE THREAD

**Issue by LARPY** [Fy-/FyPress#1]
It may be necessary to add the session processing before [https://github.com/Fy-/FyPress/blob/master/fypress/public/views.py#L67](https://github.com/Fy-/FyPress/blob/master/fypress/public/views.py#L67)?

**Comment by Fy-** [comment]
Yep, you're right. Post.views was something I did fast for templating needs, I need to rethink this. Thanks for pointing this out.
